rtl_433_mqtt_autodiscovery/run.sh

```diff
@@ -12,6 +12,9 @@ OTHER_ARGS=""
 if bashio::config.true "mqtt_retain"; then
   OTHER_ARGS="${OTHER_ARGS} --retain"
 fi
+if bashio::config.true "force_update"; then
+  OTHER_ARGS="${OTHER_ARGS} --force_update"
+fi
 
 echo "Starting rtl_433_mqtt_hass.py..."
-python3 /rtl_433_mqtt_hass.py -d -u $MQTT_USER -P $MQTT_PASS -H $MQTT_HOST -p $MQTT_PORT -R "$RTL_TOPIC" -D "$DISCOVERY_PREFIX" -i $DISCOVERY_INTERVAL $OTHER_ARGS
+python3 /rtl_433_mqtt_hass.py -d -H $MQTT_HOST -p $MQTT_PORT -R "$RTL_TOPIC" -D "$DISCOVERY_PREFIX" -i $DISCOVERY_INTERVAL $OTHER_ARGS
```

Comment:
This is a bug in `main` as well. Without `-u`, python will buffer stdout, meaning small messages won't be shown in the docker logs until the buffer is flushed, which can be a long time.

```suggestion
python3 -u /rtl_433_mqtt_hass.py -d -H $MQTT_HOST -p $MQTT_PORT -R "$RTL_TOPIC" -D "$DISCOVERY_PREFIX" -i $DISCOVERY_INTERVAL $OTHER_ARGS
```